### ISSUE TITLE
Custom URLs in navigation

### ISSUE THREAD

**Issue by Warxcell** [sulu/sulu#5082]
# Problem description

Sometimes it's necessary to have custom Urls not pointing to content or even same website. It would be convenient to have ability to add custom urls in navigation. 

# Proposed solution

To be discussed.
My proposal is to have additional admin for editing navigations.

**Comment by alexander-schranz** [comment]
@Warxcell If you have in Sulu a page in the navigation which does not point to the same website or just is not at the same level you want to have it in the navigation point.  
You create a new Page at the level you want to have it in the navigation context, go to **`Settings`** Tab and change the **`Page Type`** to **`Internal`**  or **`External`** link and add this new node the your navigation context.

**Comment by Warxcell** [comment]
@alexander-schranz good, thank you. I will create new PR for mailto: and tel: types in 
![image](https://user-images.githubusercontent.com/3340882/75340832-e0a05300-589b-11ea-88e4-1241ac089a43.png)

Also is it possible to define style to navigation item? In my specific case I need to style single item of navigation differently.

**Comment by alexander-schranz** [comment]
@Warxcell you have different option to mark the navigation item. First one you mark it by type e.g. every external nodeType will have different styling then a normal or internal one. On the other hand you still have possibility to add Tags and Excerpt Data to this External Page and you could mark this navigation item by a specified tag and style it based on the tag different.

**Comment by alexander-schranz** [comment]
@Warxcell `tel:` is now default protocol and will be available in the next release in the texteditor and url fields.

**Comment by Warxcell** [comment]
I just updated to latest master and still no `tel:`

![image](https://user-images.githubusercontent.com/3340882/77076570-a9d1de80-69fc-11ea-95fe-df0ffebba2ac.png)

**Comment by alexander-schranz** [comment]
@Warxcell when using a non tagged version of sulu you need to build the admin js yourself.

```bash
cd assets/admin
npm install
npm run build
```

If you still have problems remove all the `package-lock.json` and `node_modules` folders.

They can exist in `assets/admin` and all `vendor/sulu/sulu/src/Sulu/Bundle/*Bundle/Resources/js` folder. On unix shell I mostly just use a wildcard rm. 

```bash
cd root/folder/of/project
rm -rf ./**/node_modules
rm -rf ./**/package-lock.json 
```

As always be carefull when using `rm -rf` where you run it. 

Now you can again build your admin js as above with `npm install && npm run build` in the `assets/admin` folder.

**Comment by Warxcell** [comment]
@alexander-schranz already did that. The problem is that I don't even see it inside code?

https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/components/Url/Url.js#L30

https://github.com/sulu/sulu/blob/master/src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Url.js#L64-L67

**Comment by alexander-schranz** [comment]
@Warxcell oh its not yet back merged into master its in `release/2.0` branch. So use `2.0.*@dev` if you don't require any master features yet.

**Comment by nnatter** [comment]
I think we will merge the changes back into master soon because we also need the following PR: https://github.com/sulu/sulu/pull/5122
